chatterbot/trainers.py

```diff
@@ -136,9 +135,6 @@ class TwitterTrainer(Trainer):
     """
     Allows the chat bot to be trained using data
     gathered from Twitter.
-
-    :param random_seed_word: The seed word to be used to get random tweets from the Twitter API.
-                             This parameter is optional. By default it is the word 'random'.
```

Comment:
I though :param: docstring is used only for procedures, if it is required I'll update it also